### PR TITLE
feat(ui): rename "Edit SAM Config", improve messaging

### PR DESCRIPTION
**Pull request by justinmk3**
- Common actions like "Edit Debug Config" and "View Toolkit Logs" are not in the main AWS Explorer menu, so they are not as prominent as they should be.
- Commands are typically named in the imperative ("Do X" instead of "X Doer").
    - esp. to be consistent with other SAM-related commands
- Add some intro text in the sam launch config editor

image

image

image

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by gsweetwood — APPROVED** [review]

File: src/webviews/server.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export type OptionsToProtocol<O> = O extends WebviewCompileOptions<infer C, infe
     ? {
           submit: (result: S) => Promise<void> | void | never
           init: () => Promise<P | undefined> | P | undefined
+          getCompanyName: () => Promise<string>
```
Comment (JadenSimon) [inline src/webviews/server.ts:92]: You'll still need to call this with `this.companyName = await client.getCompanyName()` or `client.getCompanyName().then(name => this.companyName = name)` in the `created` method of `samInvokeFrontend.ts`

We could make certain properties truly global within a webview. Just haven't gotten to that yet though.

Comment (justinmk3) [reply src/webviews/server.ts:92]: yeah this push was just to check CI :)

File: src/lambda/configEditor/vue/samInvokeFrontend.ts [hunk 2]
```diff
@@ -152,6 +158,7 @@ export default defineComponent({
             if (!config) {
                 return
             }
+            const company = this.company
             this.clearForm()
```
Comment (justinmk3) [inline src/lambda/configEditor/vue/samInvokeFrontend.ts:162]: xxx: save `company` so it can be restored after clearForm()

Comment (JadenSimon) [reply src/lambda/configEditor/vue/samInvokeFrontend.ts:162]: Yeah  `getCompanyName` would be called again. This particular webview doesn't get stored in memory when not visible. If the user unfocuses the tab (or refreshes), `created` would be called again despite potentially being the same iframe. The `saveData` mixin is what persists `data` through entire reloads.

**Pull request merged**